src/esp/gfx/RenderTarget.cpp

```diff
@@ -114,6 +114,17 @@ struct RenderTarget::Impl {
 
   void renderExit() {}
 
+  void blitRgbaToDefault() {
+    GL::defaultFramebuffer.clearColor(Color4{1, 0, 0, 1});
+    GL::defaultFramebuffer
+        .clear(GL::FramebufferClear::Color | GL::FramebufferClear::Depth)
+        .bind();
+    framebuffer_.mapForRead(RgbaBuffer);
```

Comment:
I don't think mapForRead is needed either.

Reply:
Actually, this may need to stay as otherwise blit won't know which attachment to blit from.

Reply:
You're right. The mapForRead is necessary.

Reply:
Looks like you'll also need to remove https://github.com/facebookresearch/habitat-sim/blob/f8c5a1b55f90a9f9706abd104412e0a9e96d3faf/.clang-format#L2 as otherwise it'll only try to format c++

Reply:
I will open up a separate PR for formatting and linting JS.

Reply:
No idea why this comment showed up in this thread :)